Vagrantfile

```diff
@@ -9,11 +9,16 @@ $script = <<SCRIPT
 
     # Install system requirements
```

Comment:
I realize this is an old Vagrantfile...but we should be using an ansible play(s). Having the set of apt packages in here means a separate maintenance cycle if any other requirements arrive. 

That said, I think this is ok for now. Could you put a ticket in the backlog for cleanup later? (Or complete removal of the vagrantfile if we go the docker route.)